iaso/api/bulk_create_users.py

```diff
@@ -196,6 +203,14 @@ def create(self, request, *args, **kwargs):
                         )
 
                     org_units = row[csv_indexes.index("orgunit")].split(value_splitter)
+                    if self.has_user_managed_permission(request) and len(list(filter(None, org_units))) == 0:
```

Comment:
The bulk creation is now very slow (more than 20s just to create 1 user):

![slow](https://github.com/user-attachments/assets/078ceef7-e5e0-4df3-ad8b-de142fd0ee7b)

I think you shouldn't call `has_user_managed_permission` in every sub-loop because it performs database queries to check permissions.

This might be the reason why the bulk creation is now slow.

I recommend checking the rights for the connected user only once at the beginning.